### DISCUSSION TITLE
Trim the description and get only firstline

### DISCUSSION THREAD

**Discussion by buildsree** [rfennell/AzurePipelines#2006] [General]
HI,

Is it possible to trim or truncate the description and show only first line in azure wiki. since the description is too much for some tasks so we dont need dump entire description insted we do only firstline and for more description user can go and check in the task

**Description** {{lookup this.fields 'System.Description'}}

Thanks in advance

**Comment by rfennell** [comment]
Yes it is you can do it using built in [Handlebar helpers](https://github.com/helpers/handlebars-helpers), or a custom JS module where you have more flexibility. 

For the latter have a look at this [sample](https://github.com/rfennell/AzurePipelines/blob/main/SampleTemplates/XplatGenerateReleaseNotes%20(Node%20based)/Version%203%20(and%20later)/get_only_message_firstline.js).

**Comment by buildsree** [comment]
HI, 

Thank you for your replay. I am able truncate the Description and Reprosteps using below module which resolved my requirement  

module.exports = {
  lookup: function (collection, key) {
    console.log('Lookup:', collection, key);
    return collection[key];
  },
  truncate: function (str, len) {
    console.log('Truncate:', str, len);
    if (str && str.length > len) {
      return str.substring(0, len) + '...';
    }
    return str || '';
  }
};

**Reply by rfennell** [reply]
Glad you are unblocked
